posthog/api/organization_invite.py

```diff
@@ -50,6 +53,41 @@ def validate_target_email(self, email: str):
         local_part, domain = email.split("@")
         return f"{local_part}@{domain.lower()}"
 
+    def validate_private_project_access(self, private_project_access: Optional[list[dict[str, Any]]]):
```

Comment:
There's types for the params, but no type for the output of this function - I'd add it just to be consistent, but honestly not DRF expert so maybe this is expected :P